etc/api/angular_devkit/schematics/src/_golden-api.d.ts

```diff
@@ -339,6 +339,9 @@ export declare class InvalidUpdateRecordException extends BaseException {
     constructor();
 }
 
+/**
+ * @deprecated
```

Comment:
Can you please add the version when it was deprecated please? (V11)

See other deprecated methods for example 😊

Reply:
Let me know if looks fine now.